index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav class="navigation">
+            <p class="white-1 roboto-f l-height-24 f-size-18">John Doe</p>
+            <img class="menu" src="hamburguer.png" alt="">
+        </nav>
+    </header>
+
+
+    <div class="wrapper">
+
+        <div class="profile">
+            <p class="orange crete-f f-weight400 l-height-46 f-size-56">Hey There. I'm Gerson
+            <p>
+            <p class="white-1 roboto-f f-weight700 l-height-24 f-size-26">I am a software developer
+            <p>
+                <a href="https://www.linkedin.com/in/gerson-sanchez-88309b57/">
+                    <p class="white-1 popins-f l-height-24 f-size-16">I can help you build a product, feature or
+                        website. Take a
+                        look of my works. If you like what you see and have a project you need coded, don’t hesitate
+                        and
+                        contact
+                        me.
+                    </p>
+                </a>
```

Comment:
- [ ] Please note that this is a normal paragraph introduction and not a link, kindly remove the `<a>` tag since it has no purpose 🙏